tests/unit/amo/client/test_base.js

```diff
@@ -44,8 +44,8 @@ describe(__filename, () => {
 
       history.push({ pathname });
 
-      sinon.assert.calledWith(_tracking.setPage, pathname);
-      sinon.assert.calledWith(_tracking.pageView, { title: '' });
```

Comment:
I see we still have `sinon` as a dependency in package.json - is it needed for other things?